projects/packages/post-list/changelog/add-copy-post-link-action-in-posts-list

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
This adds a new feature. Isn't "minor" meant to be for minor fixes and enhancements only?

Reply:
Hm.. I'd consider this a small enhancement, but I can update..

Reply:
Note that this is a question, and not a recommendation. I'm not sure either, I haven't pushed too many changes to Jetpack since these changelogs were introduced.

Reply:
The significance field refers to patch/minor/major from https://semver.org/. In short,

* `patch`: Backward compatible bug fixes.
* `minor`: Added (or deprecated) functionality in a backwards-compatible manner.
* `major`: Broke backwards compatibility in some way.

Minor is most likely correct for a new feature, unless it's also breaking backwards compatibility in some way.